src/Tasks/GetSDKReferenceFiles.cs

```diff
@@ -1246,63 +1221,105 @@ public SdkReferenceInfo(string fusionName, string imageRuntime, bool isWinMD, bo
         /// Structure that contains the on disk representation of the SDK in memory.
         /// </summary>
         /// <remarks>This is a serialization format. Do not change member naming.</remarks>
-        [Serializable]
-        private class SDKInfo
+        internal class SDKInfo : ITranslatable
         {
-            // Current version for serialization. This should be changed when breaking changes
-            // are made to this class.
-            private const byte CurrentSerializationVersion = 1;
+            private ConcurrentDictionary<string, SdkReferenceInfo> pathToReferenceMetadata;
+            private ConcurrentDictionary<string, List<string>> directoryToFileList;
+            private int hash;
 
-            // Version this instance is serialized with.
-            private byte _serializedVersion = CurrentSerializationVersion;
+            internal SDKInfo()
+            {
+                pathToReferenceMetadata = new();
+                directoryToFileList = new();
+                hash = 0;
+            }
+
+            public SDKInfo(ITranslator translator) : this()
+            {
+                Translate(translator);
+            }
 
             /// <summary>
             /// Constructor
             /// </summary>
             public SDKInfo(ConcurrentDictionary<string, SdkReferenceInfo> pathToReferenceMetadata, ConcurrentDictionary<string, List<string>> directoryToFileList, int cacheHash)
             {
-                PathToReferenceMetadata = pathToReferenceMetadata;
-                DirectoryToFileList = directoryToFileList;
-                Hash = cacheHash;
+                this.pathToReferenceMetadata = pathToReferenceMetadata;
+                this.directoryToFileList = directoryToFileList;
+                this.hash = cacheHash;
             }
 
             /// <summary>
             /// A dictionary which maps a file path to a structure that contain some metadata information about that file.
             /// </summary>
-            public ConcurrentDictionary<string, SdkReferenceInfo> PathToReferenceMetadata { get; }
+            internal ConcurrentDictionary<string, SdkReferenceInfo> PathToReferenceMetadata { get { return pathToReferenceMetadata; } }
```

Comment:
Why is this changed from public to internal, but other properties untouched?

Reply:
I don't think this actually mattered, so I reverted it. See next comment for fiddling.